browser/services/data.js

```diff
@@ -15,6 +13,7 @@ class InstallerDataService {
     this.downloading = false;
     this.installing = false;
 
+    let env = require('remote').require('../main/env');
```

Comment:
Yes, there is another reason - when I tried to import the class into the tests I got the following:
```Cannot find module 'remote'```
This seems to have solved it.